### PR TITLE
Update sea ice layer

### PR DESCRIPTION
**Pull request by hsimonoko**
## Description

Update sea ice concentration layer to include 2021 data

## Checklist

If an item on this list is done _or not needed_, simply check it with `[x]`.

- [x] Config lockfile updated (`inv config.export > qgreenland/config/cfg-lock.json`)
- [x] Version bumped if needed (`bumpversion (major|minor|patch|prerelease|build`)
- [x] CHANGELOG.md updated
- [x] Documentation updated if needed
- [x] New unit tests if needed

### CODE REVIEW

File: qgreenland/config/cfg-lock.json [hunk 1]
```diff
@@ -1766,6 +1773,13 @@
             "ftp://sidads.colorado.edu/DATASETS/NOAA/G02135/north/monthly/geotiff/09_Sep/N_202109_concentration_v3.0.tif"
           ],
           "verify_tls": true
+        },
+        "minimum_concentration_2022": {
+          "id": "minimum_concentration_2022",
+          "urls": [
+            "ftp://sidads.colorado.edu/DATASETS/NOAA/G02135/north/monthly/geotiff/09_Sep/N_202209_concentration_v3.0.tif"
```
Comment (trey-stafford) [inline qgreenland/config/cfg-lock.json:1780]: We can add the 'max' concentration for 2022 (March) but this layer can't be added yet. September is still a little ways off :smile: .

**Review by trey-stafford — CHANGES_REQUESTED** [review]
Lets remove the September 2022 layer config since that data is not available yet.

**Review by MattF-NSIDC — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 - Updated layers:
   - "Earthquakes" updated to include 2021 data
+  - "Monthly Mean Concentration (25km)" updated to include 2021 data
```
Comment (trey-stafford) [inline CHANGELOG.md:5]: Lets note that we are specifically updating the "Feb or March (max monthly extent)" group to include 2021 data.

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,6 +7,8 @@
 
 - Updated layers:
   - "Earthquakes" updated to include 2021 data
+  - "Monthly Mean Concentration (25km) Feb or March (max monthly extent)"
```
Comment (trey-stafford) [inline CHANGELOG.md:10]: One last suggestion: I'd place a `/` in between "Monthly Mean Concentration (25km)" and "Feb or March (max monthly extent)". I would also probably include the root group "Sea ice":
> "Sea ice / Monthly Mean Concentration (25km) / Feb or March (max monthly extent)" updated to include 2021 data"

Sorry to nitpick!

**Review by trey-stafford — APPROVED** [review]

**Pull request merged**